### ISSUE TITLE
Convert resnet* and vgg19 to tensorflow and convert to tflite - the accuracy is not consistent

### ISSUE THREAD

**Issue by kyhoolee** [microsoft/MMdnn#893]
Platform (like ubuntu 16.04/win10):

Python version: 3.6

Source framework with version: Pytorch 1.5.0

Destination framework with version: Tensorflow 1.15.2

Pre-trained model path:
https://drive.google.com/drive/folders/13UW3Qw_kYxxKrICOu7dQykmq_aBJHecT?usp=sharing

Running scripts: 
https://gist.github.com/kyhoolee/1a9ae7b6f7d654e2c41d16959c200eb3

Tflite models work well on desktop - using appropriate preprocess_input with mode='torch'
But not works consistently on android devices.
Is there any ideas about the differences when converting model from pytorch to tensorflow?
